Condition expressions/Boolean operators/task.md

```diff
@@ -8,4 +8,6 @@ of `or` is `True`. The boolean operator `not` inverts the boolean expression it
 Write an expression that evaluates to `True` if `name` is equal to `"John"` and
 he is `16` or older.  
 
+Please do not add or remove any lines from the code.
```

Comment:
This seems like too strict instruction.
Maybe something like "...do not add any print statements not related to the task?"